### PR TITLE
Introduce first version of OpenShift AI extension

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: openshift-ai/runtime/src/main/java/io/quarkiverse/langchain4j/openshiftai/runtime/config/Langchain4jOpenshiftAiConfig.java [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+    URL baseUrl();
+
+    /**
+     * Timeout for BAM calls
+     */
+    @WithDefault("10s")
+    Duration timeout();
+
+    /**
+     * Whether the BAM client should log requests
```
Comment (cescoffier) [inline openshift-ai/runtime/src/main/java/io/quarkiverse/langchain4j/openshiftai/runtime/config/Langchain4jOpenshiftAiConfig.java:28]:
```suggestion
     * Whether the OpenShift AI client should log requests
```

File: openshift-ai/runtime/src/main/java/io/quarkiverse/langchain4j/openshiftai/runtime/config/Langchain4jOpenshiftAiConfig.java [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+     */
+    @WithDefault("false")
+    Boolean logRequests();
+
+    /**
+     * Whether the BAM client should log responses
```
Comment (cescoffier) [inline openshift-ai/runtime/src/main/java/io/quarkiverse/langchain4j/openshiftai/runtime/config/Langchain4jOpenshiftAiConfig.java:34]:
```suggestion
     * Whether the OpenShift AI client should log responses
```

File: openshift-ai/runtime/src/main/java/io/quarkiverse/langchain4j/openshiftai/runtime/config/Langchain4jOpenshiftAiConfig.java [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+@ConfigMapping(prefix = "quarkus.langchain4j.openshift-ai")
+public interface Langchain4jOpenshiftAiConfig {
+
+    /**
+     * Base URL where the Ollama serving is running
+     */
+    URL baseUrl();
+
+    /**
+     * Timeout for BAM calls
```
Comment (cescoffier) [inline openshift-ai/runtime/src/main/java/io/quarkiverse/langchain4j/openshiftai/runtime/config/Langchain4jOpenshiftAiConfig.java:22]:
```suggestion
     * Timeout for OpenShift AI calls
```

**Review by cescoffier — CHANGES_REQUESTED** [review]

**Comment by cescoffier** [comment]
You can remove the trust all hack as it does not work as expected.

**Comment by geoand** [comment]
Done

**Review by cescoffier — APPROVED** [review]

**Pull request merged**